wework/src/components/layout/workbenchPaneStack.module.css

```
@@ -20,3 +21,10 @@
     display: none;
   }
 }
+
+@container workbench-pane (max-height: 34rem) {
+  .paneContent :global([data-testid='desktop-chat-scroll-sticky-footer']) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)